pkgs/top-level/python-packages.nix

```
@@ -4587,6 +4587,8 @@ in {
 
   mesonpep517 = callPackage ../development/python-modules/mesonpep517 { };
 
+  MeshLabXML = callPackage ../development/python-modules/MeshLabXML { };
```

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines https://www.python.org/dev/peps/pep-0503/#normalized-names
```suggestion
  meshlabxml = callPackage ../development/python-modules/meshlabxml { };
```